static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-liveblog.scss

```diff
@@ -0,0 +1,65 @@
+$live-garnett-main-1: #dc0100;
+$live-garnett-main-2: #ffc5c2;
+
+
+.fc-container:not(.fc-container--story-package) .fc-item--type-liveblog {
```

Comment:
We totally don't want these styles to override story package styling. I also don't think I want to re-override these styles with  story package styles later in this module, i.e.
```scss
.fc-item--type-liveblog {

    //...all the live card styling, and then...

    .fc-container--story-package & {
        .fc-item__kicker {
            background-color: $news-support-7;
            color: $live-garnett-main-1;
        }

        .u-faux-block-link--hover .fc-item__kicker {
            background-color: darken($news-support-7, 5%);
        }

        .fc-trail__count--commentcount {
            color: #dadcdd;

            .inline-icon {
                fill: #dadcdd;
            }
        }
    }
```